### ISSUE TITLE
Further Explain "Infinite" As It Does Not Target Specific Audiences

### ISSUE THREAD

**Issue by hollowaykeanho** [ZORALab/Hestia#42]
# Description
```
Please provide a short description of what you have encountered below.
```
Sibert provided a good feedback that the brand's pitch "infinite possibilites" is too open so it does specfically target a group of audience. It has to be further clarified for proper targetting.

# Link
```
Please provide the documentation link(s) below.
```

# Expected Presentation
```
Please specify the expected presentation of the documentation.
```
Target audience is clear and why matters.

# Current Presentation
```
Please specify the current presentation of the documentation.
```
Not clear enough.

# Attachment
```
Please drag and drop the necessary data files (e.g. screenshot, logs, etc)
below.
```

**Comment by CoryGalyna** [comment]
> One peaceful libraries suite from developer; for Developer
Will this works?

**Comment by hollowaykeanho** [comment]
Sibert provided his feedback. Yeah. This works but needs to further nailing down the presentation and context to be extremely simple.

Like we're targeting developer but which developer? Web, backend, frontend, embedded, Go, Rust, TinGo etc. 

Also, since we're delivering in phases where each phase has its own set of target audiences, we have to consider splitting separate landing pages for them. Right now, we're on track but need to continue improving.

He also suggested to form a feedback survey team to test out the "wow effect" on the marketing side. That way, we can find more meaningful and precise content.

**Comment by CoryGalyna** [comment]
Well, I guess we just have to make `hestiaHUGO` specific to hugo communities target audiences first; and then deal with the overall product delivery.

That will work as long as we're on track. =)

**Comment by CoryGalyna** [comment]
Setting this as phase 1.1 in the project. I believe we can deliver this one easily.

**Comment by CoryGalyna** [comment]
"Frontend & Backend in One Libraries Suite; by Developers, for Developers."

**Comment by hollowaykeanho** [comment]
Ain't you a sweet heart? It's great to have you back.

**Comment by hollowaykeanho** [comment]
`Frontend+Backend in One Suite; by Deveopers; for Developers` is sharper and closer. What do you think?

**Comment by CoryGalyna** [comment]
works for me. As long as Hestia can be promoted ASAP. We have been secretly launching it but that's not how marketing work though.

> Ain't you a sweet heart? It's great to have you back.
Thanks. =)

**Comment by hollowaykeanho** [comment]
All right. I'll go for it. We still need to work out the getting started section for the marketing part. 

Let's prioritize that and then come back to optimization later.

**Comment by hollowaykeanho** [comment]
Implemented in 78e5642931d479b8bb2d3a915db6df02f4f103dd

**Comment by hollowaykeanho** [comment]
published.
